rucio-dataset-mon/kubernetes/mongo/service.yaml

```diff
@@ -0,0 +1,14 @@
+---
+apiVersion: v1
+kind: Service
+metadata:
+  name: mongodb-test
+spec:
+  type: NodePort
+  ports:
+    - port: 27017
+      protocol: TCP
+      targetPort: 27017
+      nodePort: 32000
```

Comment:
why do you need nodePort, I suggest to avoid if it is not really required and let k8s to provide routing